packages/experimental/ActivityIndicator/src/ActivityIndicator.tsx

```diff
@@ -28,26 +28,14 @@ export const ActivityIndicator = compose<ActivityIndicatorType>({
   },
   render: (props: ActivityIndicatorProps, useSlots: UseSlots<ActivityIndicatorType>) => {
     const Slots = useSlots(props);
-    const tokens = useStyling(props).root;
+    const slotProps = useStyling(props);
 
-    const memoizedActivityIndicatorData = useMemo(
-      () => ({
-        activityIndicatorColor: props.activityIndicatorColor ? props.activityIndicatorColor : tokens['activityIndicatorColor'],
-        size: props.size ? diameterSizeMap[props.size] : diameterSizeMap[tokens['size']],
-        lineThickness: props.lineThickness
-          ? lineThicknessSizeMap[props.lineThickness]
-          : props.size
-          ? lineThicknessSizeMap[props.size]
-          : lineThicknessSizeMap[tokens['lineThickness']],
-        animating: props.animating != undefined ? props.animating : true,
-        hidesWhenStopped: props.hidesWhenStopped != undefined ? props.hidesWhenStopped : true,
-      }),
-      [props.activityIndicatorColor, props.size, props.lineThickness, props.animating, props.hidesWhenStopped],
-    );
+    const animating = props.animating != undefined ? props.animating : true;
+    const hidesWhenStopped = props.hidesWhenStopped != undefined ? props.hidesWhenStopped : true;
 
     const spinAnimation = useRef(new Animated.Value(0)).current;
     useEffect(() => {
-      if (memoizedActivityIndicatorData.animating) {
+      if (animating) {
         Animated.loop(
           Animated.sequence([
```

Comment:
On my machine, I had to remove the sequence and set useNativeDriver=false in order for the animation to work properly. Is that not the case on yours?

Reply:
Starting from just this morning (which is weird since it worked yesterday), if useNativeDriver is true with the sequence, the indicator only spins once and then stops. All I have to do to get it completely working is setting useNativeDriver to false. I don't have to remove the sequence